README.md

```diff
@@ -47,6 +47,28 @@ WebNN requires a compatible browser to run, and Windows 11 v21H2 (DML 1.6.0) or
 4. In the drop-down menu, select `Enabled`. 
 5. Relaunch your browser.
 
+#### Running WebNN on NPU
+Running WebNN on NPU requires capable NPU hardware e.g. Intel® AI Boost NPU of Intel® Core™ Ultra processors.
+
+* Google Chrome Canary:
+1. Download and install the [latest Intel NPU driver](https://www.intel.com/content/www/us/en/download/794734/intel-npu-driver-windows.html).
+2. Download the latest redistributable [Microsoft.AI.DirectML](https://www.nuget.org/packages/Microsoft.AI.DirectML/), extract the package and copy the DirectML.dll (e.g. \bin\x64-win\DirectML.dll) to the Chrome Canary’s directory (e.g. C:\Users\"username"\AppData\Local\Google\Chrome SxS\Application\"version number").
+3. Launch your browser in Windows Command Line:
+```bash
+"C:\Users\\"username"\AppData\Local\Google\Chrome SxS\Application\chrome.exe" --use-redist-dml --disable_webnn_for_npu=0
```

Comment:
You may want to add a note for Chrome to explain why developers need to deploy a separate DirectML.dl and  use `--use-redist-dml` flag.